docs/src/basics.md

```diff
@@ -0,0 +1,42 @@
+# Basic usage
+
+The quickest way to get started is to install the `latest` toolchain, although this step should be automatically done if you
+installed `fuelup` via `fuelup-init`:
+
+```sh
+fuelup toolchain install latest
+```
+
+## Keeping the Sway toolchain up to date
+
+The Sway toolchain is distributed on one [release channel]: latest (with nightly being a WIP).
+`fuelup` uses the `latest` channel by default, which
+represents the latest release of the Sway toolchain.
+
+When new versions of the components within the Sway toolchain is released,
+simply type `fuelup toolchain install latest` to update:
+
+```sh
+fuelup toolchain install latest
+```
+
+## Keeping `fuelup` up to date
+
+You can request that `fuelup` update itself to the latest version of `fuelup`
+by running:
+
+```sh
+$ fuelup self update
```

Comment:
```suggestion
fuelup self update
```

We are not using `$` in other examples such as:

```sh
 fuelup toolchain install latest
```

Also linter complains about the usage of `$`: 

```
MD014/commands-show-output: Dollar signs used before commands without showing output
```
[Cool reference](https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md014---dollar-signs-used-before-commands-without-showing-output)